backend/test/images/test-run.sh

```
@@ -282,6 +287,7 @@ go run github.com/onsi/ginkgo/v2/ginkgo -r -v -p \
   -apiUrl="$API_URL" \
   -authToken="$API_TOKEN" \
   -disableTlsCheck=true \
+  -gpuVendor=$GPU_VENDOR \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)